plugins/repository-s3/build.gradle

```diff
@@ -64,14 +71,197 @@ test {
   exclude '**/*CredentialsTests.class'
 }
 
-check {
-  // also execute the QA tests when testing the plugin
-  dependsOn 'qa:amazon-s3:check'
+boolean useFixture = false
+
+String s3AccessKey = System.getenv("amazon_s3_access_key")
+String s3SecretKey = System.getenv("amazon_s3_secret_key")
+String s3Bucket = System.getenv("amazon_s3_bucket")
+String s3BasePath = System.getenv("amazon_s3_base_path")
+
+if (!s3AccessKey && !s3SecretKey && !s3Bucket && !s3BasePath) {
```

Comment:
I think if only some of these are set, this should be a configuration failure?

Reply:
fixed in f14a5f71e61